### ISSUE TITLE
MXQ Pro 4k flashing problems

### ISSUE THREAD

**Issue by brusli009** [ophub/amlogic-s9xxx-armbian#715]
Device Information 
 - SOC: S905
 - Model: MXQ-Pro 4k (think its 1G)

Armbian Version
 - Kernel Version: 5.15.79
 -  - Release: bullseye

Describe the bug
I tried flashing all of ur versions on my mxq pro but this was the only one that got anywhere, only the dtb for p201 worked to kinda boot on the version tested, i tried jammy and other versions of the bullseye but this was the only one, also tried multiple dtbs

This is where it gets

![image](https://user-images.githubusercontent.com/63052421/202915857-69da5c65-c169-4d7c-8c2e-dffc5789ce34.png)

**Comment by ophub** [comment]
https://github.com/ophub/amlogic-s9xxx-armbian/blob/838ffa2a1da8754778d4ff9219b05d192d0c92e2/build-armbian/common-files/rootfs/etc/amlogic_model_database.conf#L124

Modify the default configuration(`/boot/uEnv.txt`) and use the dedicated dtb file: `meson-gxbb-mxq-pro-plus.dtb`

**Comment by brusli009** [comment]
Thanks for the answer but i cant find the  `meson-gxbb-mxq-pro-plus.dtb` in any of those versions

**Comment by ophub** [comment]
Thank you for your feedback, there is no index in the kernel, I have submitted pr, wait for the merge, it will be available later, now you can download it in my backup file.

https://github.com/ophub/amlogic-s9xxx-armbian/tree/main/build-armbian/common-files/bootfs/dtb/amlogic

**Comment by brusli009** [comment]
I tried the dtb u linked with every version you have out there and still nothing, it shows the logo ,then says starting and bootloops after

**Comment by ophub** [comment]
https://github.com/ophub/amlogic-s9xxx-armbian/blob/main/build-armbian/common-files/bootfs/dtb/amlogic/meson-gxbb-mxq-pro-plus.dtb

**Comment by brusli009** [comment]
I tried that dtb in every build of armbian you have uploaded, some just show the s905 boot logo and then get stuck and some get to the starting kernel but boot again do the same thing

**Comment by brusli009** [comment]
I tried the versions posted in the Armbian_Aml_jammy_11.21.1016 release, and none work with neither p201 dtb or mxq pro dtb

**Comment by ophub** [comment]
Just added yesterday, download the latest firmware

**Comment by brusli009** [comment]
Sadly with the jammy build i either got a black screen after the logo or was stuck at the logo. With the bulseye 5.10 im getting the error, while on 5.15 it was stuck on the logo
Maybe there is a older version when you first supported this device that will work
![image](https://user-images.githubusercontent.com/63052421/203272573-99eaceb1-4d8f-4a99-8dc1-9353bb8b3552.png)

**Comment by ophub** [comment]
```
ls /boot/dtb/amlogic/meson-gxbb-mxq-pro-plus.dtb
cat /boot/uEnv.txt
```

**Comment by brusli009** [comment]
where should these be added?

**Comment by ophub** [comment]
Instead of adding, you execute these 2 commands in the terminal to see the result. I look at your configuration.

**Comment by brusli009** [comment]
i only get to the console in bulseye 5.15 when using the p201 dtb but after entering the commands this is what i get
![image](https://user-images.githubusercontent.com/63052421/203278026-01714194-90c9-4c62-8eed-9578dd87da6f.png)

**Comment by ophub** [comment]
Download today's firmware, write to USB.
Insert the USB into the computer, open the boot partition, modify the `uEnv.txt` inside, change the value of dtb to `meson-gxbb-mxq-pro-plus.dtb`, save it and insert it into MXQ to start.

**Comment by ophub** [comment]
You can't understand what I'm saying and can't continue.
This is a very simple operation, you read the steps I mentioned above carefully.
This device has a dedicated dtb that you can use if you modify it correctly.
But for now you need to stop and read the operating instructions first.

**Comment by brusli009** [comment]
Sorry to say but i am doing instructions corectly but it does not boot

**Comment by brusli009** [comment]
![image](https://user-images.githubusercontent.com/63052421/203280818-2b3fd25f-198e-4e02-aca6-ba794f2ef4e7.png)
this is the uEnv.txt but it wont boot

**Comment by ophub** [comment]
Enter the boot/dtb/amlogic directory, is there a meson-gxbb-mxq-pro-plus.dtb file?

**Comment by brusli009** [comment]
Yes there is, tried it in jammy latest realese and bullseye latest, most just show the s905 logo and dont boot
![image](https://user-images.githubusercontent.com/63052421/203283980-5f1ed825-55fc-4baf-acc8-e069edc16468.png)

**Comment by ophub** [comment]
Then why does the picture you posted above indicate that there is no file? You plug this into the box and reboot.

**Comment by brusli009** [comment]
I have been able to boot to armbian using a 32gb sd card with a sd to usb adapter and using the 5.15 bulseye with the p201.dtb tho wifi isnt working and it dosent display everytime through hdmi, also i cant install it on the storage 
![image](https://user-images.githubusercontent.com/63052421/203303404-c0fad961-6032-4c2c-a1f6-0acf79c08eeb.png)

**Comment by ophub** [comment]
The documentation on the home page states that s905 does not support writing to emmc, and writing will turn into bricks.

**Comment by korzhan** [comment]
![image](https://user-images.githubusercontent.com/47418633/226171930-7f20c2c2-22f1-42cd-99c2-e25619b64e16.png)

**Comment by korzhan** [comment]
![image](https://user-images.githubusercontent.com/47418633/226171948-c0108f6e-65b3-4757-b8d8-16505b517415.png)
I can't connect to wifi. Already tried everything. MXQ Pro Plus 4K 905.

**Comment by ophub** [comment]
wait next kernel

**Comment by ToShukKr** [comment]
can any help me to boot ? what we need ? 
only insert sdcard with armbian and hold reset button in AV and turn on a power ?

what happend with incorrect dbd? i have a looped reboot on TVBOX logo (not armbian)

**Comment by khozaimi** [comment]
> I have been able to boot to armbian using a 32gb sd card with a sd to usb adapter and using the 5.15 bulseye with the p201.dtb tho wifi isnt working and it dosent display everytime through hdmi, also i cant install it on the storage ![image](https://user-images.githubusercontent.com/63052421/203303404-c0fad961-6032-4c2c-a1f6-0acf79c08eeb.png)

can you explain this? step by step installation Openwrt on MXQ Pro 4K. Thank you
